src/docfx/serve/Serve.cs

```diff
@@ -36,15 +36,19 @@ void Configure(IApplicationBuilder app)
 
         private static async Task StartLanguageServer(HttpContext context, CommandLineOptions options, Package? package)
         {
-            if (context.WebSockets.IsWebSocketRequest)
+            // The SynchronizationContext is lost here, verbose needs to be reset
```

Comment:
Technically this is called execution context not synchronization context.
```suggestion
            // The execution context is lost here, verbose needs to be reset
```